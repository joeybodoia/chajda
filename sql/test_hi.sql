CREATE EXTENSION IF NOT EXISTS pspacy;
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_tsvector('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"');
