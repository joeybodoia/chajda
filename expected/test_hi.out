CREATE EXTENSION IF NOT EXISTS pspacy;
NOTICE:  extension "pspacy" already exists, skipping
SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                    spacy_lemmatize                                    
---------------------------------------------------------------------------------------
   उदाहरण:4 परीक्षण:5 वाक्य:6 परीक्षण:10 मामलों:11 उत्पन्न:13 google:17 अनुवाद:18 डाल:20 हूं:22
(1 row)

SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                      spacy_lemmatize                       
------------------------------------------------------------
   उदाहरण परीक्षण वाक्य परीक्षण मामलों उत्पन्न google अनुवाद डाल हूं
(1 row)

SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                                  spacy_lemmatize                                                                   
----------------------------------------------------------------------------------------------------------------------------------------------------
   यह:2 मेरा:3 उदाहरण:4 परीक्षण:5 वाक्य:6 है:7 जो:8 मैं:9 परीक्षण:10 मामलों:11 को:12 उत्पन्न:13 करने:14 के:15 लिए:16 google:17 अनुवाद:18 में:19 डाल:20 रहा:21 हूं:22
(1 row)

SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                       spacy_lemmatize                                       
---------------------------------------------------------------------------------------------
   यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए google अनुवाद में डाल रहा हूं
(1 row)

SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                           spacy_lemmatize                                           
-----------------------------------------------------------------------------------------------------
   ":2 उदाहरण:5 परीक्षण:6 वाक्य:7 परीक्षण:11 मामलों:12 उत्पन्न:14 google:18 अनुवाद:19 डाल:21 हूं:23 ।:24 ":25
(1 row)

SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                         spacy_lemmatize                          
------------------------------------------------------------------
   " उदाहरण परीक्षण वाक्य परीक्षण मामलों उत्पन्न google अनुवाद डाल हूं । "
(1 row)

SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                          spacy_lemmatize                                                                          
-------------------------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 यह:3 मेरा:4 उदाहरण:5 परीक्षण:6 वाक्य:7 है:8 जो:9 मैं:10 परीक्षण:11 मामलों:12 को:13 उत्पन्न:14 करने:15 के:16 लिए:17 google:18 अनुवाद:19 में:20 डाल:21 रहा:22 हूं:23 ।:24 ":25
(1 row)

SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                          spacy_lemmatize                                          
---------------------------------------------------------------------------------------------------
   " यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए google अनुवाद में डाल रहा हूं । "
(1 row)

SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                    spacy_lemmatize                                    
---------------------------------------------------------------------------------------
   उदाहरण:4 परीक्षण:5 वाक्य:6 परीक्षण:10 मामलों:11 उत्पन्न:13 Google:17 अनुवाद:18 डाल:20 हूं:22
(1 row)

SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                      spacy_lemmatize                       
------------------------------------------------------------
   उदाहरण परीक्षण वाक्य परीक्षण मामलों उत्पन्न Google अनुवाद डाल हूं
(1 row)

SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                                  spacy_lemmatize                                                                   
----------------------------------------------------------------------------------------------------------------------------------------------------
   यह:2 मेरा:3 उदाहरण:4 परीक्षण:5 वाक्य:6 है:7 जो:8 मैं:9 परीक्षण:10 मामलों:11 को:12 उत्पन्न:13 करने:14 के:15 लिए:16 Google:17 अनुवाद:18 में:19 डाल:20 रहा:21 हूं:22
(1 row)

SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                       spacy_lemmatize                                       
---------------------------------------------------------------------------------------------
   यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं
(1 row)

SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                           spacy_lemmatize                                           
-----------------------------------------------------------------------------------------------------
   ":2 उदाहरण:5 परीक्षण:6 वाक्य:7 परीक्षण:11 मामलों:12 उत्पन्न:14 Google:18 अनुवाद:19 डाल:21 हूं:23 ।:24 ":25
(1 row)

SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                         spacy_lemmatize                          
------------------------------------------------------------------
   " उदाहरण परीक्षण वाक्य परीक्षण मामलों उत्पन्न Google अनुवाद डाल हूं । "
(1 row)

SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                          spacy_lemmatize                                                                          
-------------------------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 यह:3 मेरा:4 उदाहरण:5 परीक्षण:6 वाक्य:7 है:8 जो:9 मैं:10 परीक्षण:11 मामलों:12 को:13 उत्पन्न:14 करने:15 के:16 लिए:17 Google:18 अनुवाद:19 में:20 डाल:21 रहा:22 हूं:23 ।:24 ":25
(1 row)

SELECT spacy_lemmatize('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                          spacy_lemmatize                                          
---------------------------------------------------------------------------------------------------
   " यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं । "
(1 row)

SELECT spacy_tsvector('hi',' "यह मेरा उदाहरण परीक्षण वाक्य है जो मैं परीक्षण मामलों को उत्पन्न करने के लिए Google अनुवाद में डाल रहा हूं।"');
                                         spacy_tsvector                                         
------------------------------------------------------------------------------------------------
 'google':17 'अनुवाद':18 'उत्पन्न':13 'उदाहरण':4 'डाल':20 'परीक्षण':5,10 'मामलों':11 'वाक्य':6 'हूं':22
(1 row)

